process/gasCost.go

```diff
@@ -28,6 +28,8 @@ type BuiltInCost struct {
 	ESDTNFTAddUri            uint64
 	ESDTNFTUpdateAttributes  uint64
 	ESDTNFTMultiTransfer     uint64
+	TrieLoad                 uint64
```

Comment:
a little bit misleading name. This is actually `TrieLoadPerLeaf` cost. Also the next Store cost.

Reply:
Renamed